res/meson.build

```diff
@@ -68,3 +68,4 @@ subdir('items')
 subdir('pokemon')
 subdir('sound')
 subdir('trainers')
+subdir('data')
```

Comment:
**polish**: Keep this list sorted in lexicographical order, unless a specific processing order is needed.